src/plugins/intel_gpu/tests/functional/single_layer_tests/dynamic/space_to_depth.cpp

```diff
@@ -0,0 +1,195 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "shared_test_classes/single_layer/space_to_depth.hpp"
+#include "shared_test_classes/base/ov_subgraph.hpp"
+#include "ie_precision.hpp"
+#include "ov_models/builders.hpp"
+#include "common_test_utils/ov_tensor_utils.hpp"
+#include <string>
+
+using namespace ov::op::v0;
+using namespace ov::test;
+
+namespace GPULayerTestsDefinitions {
+
+typedef std::tuple<
+    InputShape,                         // Input shape
+    ElementType,                        // Input element type
+    SpaceToDepth::SpaceToDepthMode,     // Mode
+    std::size_t                         // Block size
+> SpaceToDepthLayerGPUTestParams;
+
+class SpaceToDepthLayerGPUTest : public testing::WithParamInterface<SpaceToDepthLayerGPUTestParams>,
+                                 virtual public ov::test::SubgraphBaseTest {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<SpaceToDepthLayerGPUTestParams> obj) {
+        InputShape shapes;
+        ElementType inType;
+        SpaceToDepth::SpaceToDepthMode mode;
+        std::size_t blockSize;
+        std::tie(shapes, inType, mode, blockSize) = obj.param;
+
+        std::ostringstream results;
+        results << "IS=" << ov::test::utils::partialShape2str({shapes.first}) << "_";
+        results << "TS=";
+        for (const auto& item : shapes.second) {
+            results << ov::test::utils::vec2str(item) << "_";
+        }
+        results << "Prc=" << inType << "_";
+        switch (mode) {
+            case SpaceToDepth::SpaceToDepthMode::BLOCKS_FIRST:
+                results << "BLOCKS_FIRST_";
+                break;
+            case SpaceToDepth::SpaceToDepthMode::DEPTH_FIRST:
+                results << "DEPTH_FIRST_";
+                break;
+            default:
+                throw std::runtime_error("Unsupported SpaceToDepthMode");
+        }
+        results << "BS=" << blockSize;
+
+        return results.str();
+    }
+
+protected:
+    void SetUp() override {
+        InputShape shapes;
+        SpaceToDepth::SpaceToDepthMode mode;
+        std::size_t blockSize;
+        std::tie(shapes, inType, mode, blockSize) = this->GetParam();
+
+        targetDevice = ov::test::utils::DEVICE_GPU;
+        init_input_shapes({shapes});
+
+        ov::ParameterVector params;
+        for (auto&& shape : inputDynamicShapes)
+            params.push_back(std::make_shared<ov::op::v0::Parameter>(inType, shape));
+
+        auto d2s = std::make_shared<ov::op::v0::SpaceToDepth>(params[0], mode, blockSize);
+
+        ngraph::ResultVector results;
+        for (size_t i = 0; i < d2s->get_output_size(); i++)
+            results.push_back(std::make_shared<ngraph::opset1::Result>(d2s->output(i)));
+        function = std::make_shared<ngraph::Function>(results, params, "SpaceToDepth");
```

Comment:
please, move all `ngraph::` usage to `ov::`

Reply:
It is Fixed.

Reply:
Please, replace `ngraph::Function` with `ov::Model`

Reply:
@byungilm  please check comment. It is not fixed.

Reply:
Sorry, I fixed Result to ov, but missed Function. It is fixed.

Reply:
@vurusovs sorry, all fixed. please check the compare of [37be548],[d332142]